File: JAVA-Basic/JavaBasic/src/ConditionalStatements/ConditionalStatementsMoreExercises/PipesInPool.java
package ConditionalStatements.ConditionalStatementsMoreExercises;

import java.util.Scanner;

public class PipesInPool {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int V = Integer.parseInt(scan.nextLine());
        int p1 = Integer.parseInt(scan.nextLine());
        int p2 = Integer.parseInt(scan.nextLine());
        double h = Double.parseDouble(scan.nextLine());

        double totalLitres = (p1 * h) + (p2 * h);
        double percentFromP1 = (p1*h / totalLitres) * 100;
        double percentFromP2 = (p2*h / totalLitres) * 100;

        if (totalLitres > V) {
            System.out.printf("For %.2f hours the pool overflows with %.2f liters.", h, totalLitres - V);
        } else {
            System.out.printf("The pool is %.2f%% full. Pipe 1: %.2f%%. Pipe 2: %.2f%%.", (totalLitres / V)*100, percentFromP1, percentFromP2);
        }
    }
}

File: 01.JS-Basic/More Exercises/PB - More Exercises/Conditional Statements/PipesInPool.js
function pipesInPool(input) {

    let v = Number(input[0])    // V – Обем на басейна в литри 
    let P1 = Number(input[1])    // P1 – дебит на първата тръба за час 
    let P2 = Number(input[2])   // P2 – дебит на втората тръба за час
    let H = Number(input[3])    // H – часовете които работникът отсъства 

    let P1fill = P1 * H
    let P2fill = P2 * H
    let poolFill = (P1 + P2) * H

    if (poolFill > v) {

        console.log(`For ${H} hours the pool overflows with ${(poolFill - v)} liters.`)
    }
    else { console.log(`The pool is ${(poolFill / v * 100).toFixed(2)}% full. Pipe 1: ${(P1fill / poolFill * 100).toFixed(2)}%. Pipe 2: ${(P2fill / poolFill * 100).toFixed(2)}%.`) }

}
pipesInPool(["1000", "100", "120", "3"])